gc/base/standard/Scavenger.cpp

```diff
@@ -4629,6 +4629,10 @@ MM_Scavenger::percolateGarbageCollect(MM_EnvironmentBase *env,  MM_MemorySubSpac
 	/* Set last percolate reason */
 	_extensions->heap->getPercolateStats()->setLastPercolateReason(percolateReason);
 
+	if (CRITICAL_REGIONS == percolateReason) {
```

Comment:
put a comment here, something like this:
Percolate Global due to Critical regions was not due to tight (Tenure) heap, hence should not really affect its resizing.